Makefile

```diff
@@ -27,40 +27,27 @@ PYTEST_ARGS ?=
 BUILD_IMAGES ?= 1
 
 ifdef KEEP_TEST_CONTAINER
-  DOCKER_RM_CONTAINER =
+  CONTAINER_RM =
 else
-  DOCKER_RM_CONTAINER = --rm
+  CONTAINER_RM = --rm
 endif
 
-# Let the user specify DOCKER at the CLI, otherwise try to autodetect a working podman or docker
-ifndef DOCKER
-  DOCKER := $(shell podman run --rm alpine echo podman 2> /dev/null)
-  ifndef DOCKER
-    DOCKER := $(shell docker run --rm alpine echo docker 2> /dev/null)
-    ifndef DOCKER
-      DUMMY := $(warning Many of the make targets require a working podman or docker.  Please install one of those and check that `podman run alpine echo hello` or `docker run alpine echo hello` work)
-    endif
-  endif
+# Let the user specify PODMAN at the CLI, otherwise try to autodetect a working podman
+ifndef PODMAN
+  PODMAN := $(shell podman run --rm alpine echo podman 2> /dev/null)
+	ifndef PODMAN
+		DUMMY := $(warning podman is not detected. Majority of commands will not work. Please install and verify that podman --version works.)
+	endif
```

Comment:
I think the indentation is a bit off, but nevertheless, it works just fine.

Reply:
Omg it's using tabs not spaces.. I have to fix this

Reply:
Oh nvm you have to use tabs for Makefile. But I still messed it up so fixing it now